dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -54,6 +54,11 @@
 .. warning::
     This class is experimental and its API might change without notice.
 
+.. note::
+    Setting ``no_copy`` on the external source nodes when defining the pipeline is considered
```

Comment:
Is this level of detail OK or too much?

Reply:
Do we need this option if it is `enabled automatically if possible`?

Reply:
This option is present in the external_source operator. I really can't prohibit user from setting it before he passes the Pipeline to the Dataset, so they might think, that setting it will do something (make stuff faster?).

Reply:
Make sense.

Reply:
I think this information should be added to the documentation of `no_copy` argument in `ExternalSource`.